resources/style.md

```diff
@@ -33,3 +33,10 @@ Individual projects should configure `flake8` to suit their needs, for example t
 Git commit messages should convey the change succinctly, but with enough detail to be understood without extra context.
 
 See the [Git Methodology]({% link resources/git.md %}#git-commits) for further details.
+
+## GitHub Pull Requests
+
+Draft Pull Requests should be opened while work is in progress. This lets other
+developers know draft changes are coming without initialing a the review process
+immediately. Once the pull request is ready, click "Ready for review" to get
+feedback on the changes.
```

Comment:
```suggestion
## GitHub Pull Requests

[Draft pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests) should be opened while work is in progress. This prevents review notifications from being sent to code owners before a pull request is ready to be reviewed. Once the pull request is ready, click "Ready for review" to get
feedback on the changes.
```

Few minor suggestions.

I added a link to GitHub's draft pull request doc section [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests).

Also tweaked the second sentence a bit to mention that notifications aren't sent to code owners for draft PRs. I think that part is important.

Might also be worth mentioning that GHAs will still run on draft PRs, but I'll leave that up to you.